releasenotes.txt

```diff
@@ -3,16 +3,20 @@ spatially enabled database and manipulate geometries in
 HQL or in .NET code providing you with a fully integrated
 GIS programming experience.
 
-Read documentation at NH Spatial wiki site:
-http://www.nhforge.org/wikis/spatial/default.aspx
+Read documentation at NH Spatial site:
+http://nhibernate.info/doc/spatial/introduction.html
 
 Please submit your bug reports, feature requests or 
 patches to NH issue tracker:
-http://jira.nhibernate.org/browse/NHSP
+https://nhibernate.jira.com/projects/NHSP

```

Comment:
The JIRA tracker is still live, apparently not migrated to GitHub issues, and accepting new issues (not moved to the "archived" permission scheme). So I have just fix the URL without changing it to GitHub issues.